email/parse.c

```diff
@@ -1379,6 +1379,17 @@ struct Body *mutt_read_mime_header(FILE *fp, bool digest)
         mutt_str_replace(&p->description, c);
         rfc2047_decode(&p->description);
       }
+      else if (mutt_istr_equal("id", line + plen))
+      {
+        // strip <angle braces> from Content-ID: header
```

Comment:
Is it OK to assume there are `<>`s?
Should we check?

Reply:
It can't harm to check. Can it be assumed that there is no leading or trailing white space at this point?